### PR TITLE
Yelp (Independent Publisher)

### PR DESCRIPTION
**Pull request by ahmadn82**
---
When submitting a connector, please check the following conditions for your PR to ensure a smooth certification process.

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 

image

image

image

### CODE REVIEW

**Comment by Laskewitz** [comment]
Nice one @ahmadn82 !

File: independent-publisher-connectors/Yelp/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,1238 @@
...
+                      },
+                      "phone": {
+                        "type": "string",
+                        "description": "Phone number of the business.",
+                        "title": "Phone",
+                        "x-ms-visibility": "important"
+                      },
+                      "id": {
+                        "type": "string",
+                        "description": "Unique Yelp ID of this business. Example: '4kMBvIEWPxWkWKFN__8SxQ'",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Yelp/apiDefinition.swagger.json:60]: @ahmadn82 Can you please remove secret-look-alike value from the description as it get security alerts

File: independent-publisher-connectors/Yelp/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,1238 @@
...
+                        "description": "Unique Yelp alias of this business. Can contain unicode characters. Example: \"yelp-san-francisco\"",
+                        "title": "Alias",
+                        "x-ms-visibility": "advanced"
+                      },
+                      "is_closed": {
+                        "type": "boolean",
+                        "description": "Whether business has been (permanently) closed",
+                        "title": "Is Closed",
+                        "enum": [
+                          "",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Yelp/apiDefinition.swagger.json:75]: @ahmadn82 Can you please let us know how an empty string is allowed in a Boolean enum ?

File: independent-publisher-connectors/Yelp/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,1238 @@
...
+                        },
+                        "description": "user"
+                      },
+                      "text": {
+                        "type": "string",
+                        "description": "text"
+                      },
+                      "time_created": {
+                        "type": "string",
+                        "description": "time_created"
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Yelp/apiDefinition.swagger.json:1050]: @ahmadn82 All the description values of the properties are expected to be free from grammatical and spelling errors. Can you please provide valid descriptions.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: independent-publisher-connectors/Yelp/apiDefinition.swagger.json (hunk 1 above)
Comment (ahmadn82) [reply independent-publisher-connectors/Yelp/apiDefinition.swagger.json:60]: @sriyen-msft Yes I can... Done!

File: independent-publisher-connectors/Yelp/apiDefinition.swagger.json (hunk 2 above)
Comment (ahmadn82) [reply independent-publisher-connectors/Yelp/apiDefinition.swagger.json:75]: @sriyen-msft I can't TBH... Probably copied it by mistake, as I tend to work on raw swagger file 😉

File: independent-publisher-connectors/Yelp/apiDefinition.swagger.json (hunk 3 above)
Comment (ahmadn82) [reply independent-publisher-connectors/Yelp/apiDefinition.swagger.json:1050]: @sriyen-msft Fixed!

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**